documentation/modules/rn-2.5.adoc

```diff
@@ -0,0 +1,180 @@
+
+.Non-supported guest operating systems in warm migrations
+
+Warm migrations and migrations to remote OCP clusters from vSphere do not support all types of guest operating systems that are supported in cold migrations to the local OCP cluster. It is a consequence of using RHEL 8 in the former case and RHEL 9 in the latter case. +
+See link:https://access.redhat.com/articles/1351473[Converting virtual machines from other hypervisors to KVM with virt-v2v in RHEL 7, RHEL 8, and RHEL 9] for the list of supported guest operating systems.
+
+.VMs from vSphere with RHEL 9 guest operating system may start with network interfaces that are down
+
+When migrating VMs that are installed with RHEL 9 as guest operating system from vSphere, their network interfaces could be disabled when they start in {ocp-name} Virtualization. link:https://issues.redhat.com/browse/MTV-491[(MTV-491)]
+
+////
+
+Going to move to a new section called UPGRADE NOTES
+.Upgrade from 2.4.0 fails
+
+When upgrading from MTV 2.4.0 to a later version, the operation fails with an error that says the field 'spec.selector' of deployment `forklift-controller` is immutable. Workaround: remove the custom resource `forklift-controller` of type `ForkliftController` from the installed namespace, and recreate it. The user needs to refresh the OCP Console once the `forklift-console-plugin` pod runs to load the upgraded {project-short} web console. link:https://issues.redhat.com/browse/MTV-518[(MTV-518)]
+////
+
+For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/browse/MTV-562?filter=12419159[Known Issues] in Jira.
+
+[id="resolved-issues-25_{context}"]
+== Resolved issues
+
+// are there any resolved issues you want to highlight in this release?
+
+.Snapshots that are created during the migration in OpenStack are not deleted
+
+In previous releases of {project-short}, the Migration Controller service did not delete snapshots that were created during the migration for source virtual machines in OpenStack automatically.
+
+This issue is resolved in {project-short} {project-version}, all the snapshots created during the migration are removed after the migration has been completed successfully. link:https://issues.redhat.com/browse/MTV-620[MTV-620]
+
+.{rhv-short} snapshots are not deleted after a successful migration
+
+In previous releases of {project-short}, the Migration Controller service did not delete snapshots automatically after a successful warm migration of a VM from {rhv-short}.
+
+This issue is resolved in {project-short} {project-version}, the snapshots generated during migration are removed after a successful migration, and the original snapshots are not removed after a successful migration. link:https://issues.redhat.com/browse/MTV-349[(MTV-349)]
+
+.Migration fails during precopy/cutover while a snapshot operation is executed on the source VM
```

Comment:
ok, this item should change:
1. we need to keep a known issue that is correlated with https://issues.redhat.com/browse/MTV-456 and says that warm migration fails when a snapshot is created on the source VM not by MTV (snapshot that is created in RHV by the user / other tool)
2. this item should include two parts: (a) Cutover can not be triggered when there's an ongoing precopy operation (https://issues.redhat.com/browse/MTV-686); and if https://issues.redhat.com/browse/MTV-687 gets verified for 2.5.0 then (b) Warm migration doesn't fail when an operation that locks the VM is executed in RHV.

about (2), we can split this item to two items or combine them under a single one, it's title can be something like "Improve warm migrations from RHV" or maybe more specifically "Resolved failures of warm migrations from RHV due to timing issues"

Reply:
[MTV-686](https://issues.redhat.com/browse/MTV-686) is on verified - add as **Resolved Issue**?

[MTV-687](https://issues.redhat.com/browse/MTV-687) is on verified - add as **Resolved Issue**?

Reply:
Added [MTV-686](https://issues.redhat.com/browse/MTV-686) as a resolved issue as the status is VERIFIED

Will give [MTV-687](https://issues.redhat.com/browse/MTV-687) a day or two to see if moves to verified